editor/editor_asset_installer.cpp

```diff
@@ -107,6 +107,7 @@ void EditorAssetInstaller::open(const String &p_path, int p_depth) {
 		files_sorted.insert(name);
 
 		ret = unzGoToNextFile(pkg);
+		ERR_BREAK(ret != UNZ_EOF && ret != UNZ_OK);
```

Comment:
This defeats the purpose of the conditional in the `while` loop. What warning is this handling?